lib/puppet/reference/configuration.rb

```diff
@@ -22,10 +22,16 @@
 
     # Now print the data about the item.
     val = object.default
-    if name.to_s == "vardir"
-      val = "/opt/puppetlabs/puppet/cache"
-    elsif name.to_s == "confdir"
-      val = "/etc/puppetlabs/puppet"
+    if name.to_s == 'vardir'
+      val = 'Unix/Linux: /opt/puppetlabs/puppet/cache -- Windows: C:\ProgramData\PuppetLabs\puppet\cache -- Non-root user: ~/.puppetlabs/opt/puppet/cache'
```

Comment:
I've been avoiding doing that, on the assumption that most Windows users DON'T understand that just fine... but that assumption might have been bad. I would kind of want to check with support or something, unless the whole windows team agrees on this?